service/windows/installer/custom-actions/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
```

Comment:
And this create must also be formatted and added to clippy.

Reply:
Done.